Sources/ScriptParse/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(ScriptParse
```

Comment:
It's great in general that you added CMake support here — I wonder if it is needed for this particular feature, though, because the CMake support is only used during bootstrapping, i.e. building enough of SwiftPM so that it can build itself.  We could probably simplify but not including this feature during the bootstrapping?  That would safe SwiftSyntax from having to be made to bootstrap.

Reply:
This directory is currently excluded from bootstrapping.